### PR TITLE
Add my name and image + Change some stuff

### LINKED ISSUE #479: Students after 'Rahul Arora' not visible
**Actual Behaviour**

Even though there are names and data after Rahul Arora in "students.yml", these are not reflected on the website
**Expected Behaviour**
The students should show up on the section

**Steps to reproduce it**
Go to gci18.fossasia.org/#students

you will notice the last name to be of "Rahul Arora", even though in students.yml it's not the last name

**LogCat for the issue**

**Screenshots of the issue**

**Would you like to work on the issue?**

### PR DESCRIPTION
**Pull request by namphong124563**
Demo site link: namphong124563.github.io

- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://namphong124563.github.io/)

- Fixes #479 

## Description / Changes

- Redirect default image of students and mentors
- Remove limit in displaying students and mentors (for further features)

### Before:
image

### After:

image

- - - - - - - - - - - -

### CODE REVIEW

**Comment by namphong124563** [comment]
But he only changes from 30 to 9999, and I don't think that's gonna work .-.

**Comment by namphong124563** [comment]
Sorry for that, I forgot about that I'm in a global community

File: _data/students.yml [hunk 1]
```diff
@@ -160,3 +160,10 @@
 - name: Yash Garg
   github: Yash-Garg
   image: yashg.png
+  
+- name: 三銃士
+  github: namphong124563
+  image: namphong124563.jpeg
+  facebook: namphong124563
```
Comment (sarnava1) [inline _data/students.yml]: Please remove these extra lines. Only give your name, image and github link.

Comment (namphong124563) [reply _data/students.yml]: Finished.

**Comment by sarnava1** [comment]
Now merge the commits

**Comment by sarnava1** [comment]
You can solve the other issue by claiming the task in GCI dashboard which aims to fix issues in the GCI website.

**Comment by namphong124563** [comment]
> Now merge the commits

I don't familiar with Git Bash, so how to merge the commits? I have searched many sites but I still can not merge them into one and edit.

**Comment by abishekvashok** [comment]
If you can't squash, we do have the squahs and merge option :)

**Comment by sarnava1** [comment]
Please change the PR title to what you di and resolve the conflicts

**Comment by namphong124563** [comment]
Can you guys delete or remove this PR so I can post another one? Because I don't know what to do now and I guess this is a mess and I shouldn't make it messier :(

**Comment by namphong124563** [comment]
I will make two separated PR, one for adding my data and one for solving minor bugs later, so this can be put behind and you guys don't have to waste more time.

**Pull request closed without merging**